packages/components/src/scrollable/stories/index.js

```diff
@@ -1,12 +1,12 @@
 /**
  * Internal dependencies
  */
-import { View } from '../../../view';
-import { Scrollable } from '../index';
+import { View } from '../../view';
+import { Scrollable } from '../';
 
 export default {
 	component: Scrollable,
-	title: 'G2 Components (Experimental)/Scrollable',
+	title: 'Components (Experimental)/Scrollable',
```

Comment:
I think it would have been useful if the story allowed us to change the scroll direction and toggle the smooth scroll option.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32446/commits/dcaafa77a1bca0e641c42ee8c334d901394129d2